mingw-w64-git/PKGBUILD

```diff
@@ -50,7 +50,9 @@ build() {
   export PYTHON_PATH=/usr/bin/python2
   cd "$srcdir/git"
 
-  make -j1 all &&
```

Comment:
Funny you mention the "hung without explanation" - using `-j` without any additional parameters (i.e. number of jobs) causes my machine to hang!